src/grt/grt-wave_opt_file.adb

```diff
@@ -42,21 +43,27 @@ package body Grt.Wave_Opt_File is
          when Warning =>
             Report_C ("warning: ");
       end case;
-      Report_C ("in file '");
       Report_C (File_Path.all);
-      Report_C ("' at line ");
-      To_String (Lineno_Str, First, Ghdl_I32 (Line_Context.Num));
-      Report_C (Lineno_Str (First .. Lineno_Str'Last));
-      Report_C (" - ");
-      Report_C (Line_Context.Str.all);
-      Report_C (" : ");
+      Report_C (":");
+      To_String (Value_Str, First, Ghdl_I32 (Line_Pos));
```

Comment:
Fixed


Le 27. 07. 16 à 05:17, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file.adb 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72375071>:
>
> >        Report_C (File_Path.all);
> > -      Report_C ("' at line ");
> > -      To_String (Lineno_Str, First, Ghdl_I32 (Line_Context.Num));
> > -      Report_C (Lineno_Str (First .. Lineno_Str'Last));
> > -      Report_C (" - ");
> > -      Report_C (Line_Context.Str.all);
> > -      Report_C (" : ");
> > +      Report_C (":");
> > +      To_String (Value_Str, First, Ghdl_I32 (Line_Pos));
>
> No need to convert: there is a Report_C for Natural, so:
> Report_C (Line_Pos);
> should work
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/77c854621cce3da7afdd08837584aea3fe27c1ca#r72375071>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePunQXKj7ztiXt-i6GqjKMsfITadcks5qZs3HgaJpZM4JVgTD>.
>